### PR TITLE
add reputation v0

### PR DESCRIPTION
**Pull request by 0xTail**

### CODE REVIEW

File: packages/reputation/test/util.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+module.exports = {
+    // formats result from PullReputationScore into a dict
+    cleansePullReputationScore: async promise => {
+        let thrown = undefined;
+        try {
+            let result = await promise;
+            return {reviewTotal: result[0],
+            reviewCount: result[1],
+            personalTotal: result[2],
```
Comment (tc) [inline packages/reputation/test/util.js]: what's `personal` mean?

File: packages/reputation/test/util.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            personalCount: result[3],
+            bookingTotal: result[4],
+            bookingCount: result[5],
+            miscTotal: result[6],
```
Comment (tc) [inline packages/reputation/test/util.js]: same here, what does `misc` refer to?

File: packages/reputation/test/util.js (hunk 1 above)
Comment (0xTail) [reply packages/reputation/test/util.js]: This is referring to:
`
        signalTypeMapping[0x0] = 'review';   // e.g., host provides review; guest provides a review
        signalTypeMapping[0x1] = 'personal'; // e.g., when a user updates their profile, adds number, etc
        signalTypeMapping[0x2] = 'booking';  // e.g., when a user makes, cancels, rejects a booking
        signalTypeMapping[0x3] = 'misc';     // miscellanous items
`

File: packages/reputation/test/util.js (hunk 2 above)
Comment (0xTail) [reply packages/reputation/test/util.js]: see previous comment

File: packages/reputation/test/util.js (hunk 1 above)
Comment (tc) [reply packages/reputation/test/util.js]: I see, so a user adds to their `reputation` when they add more `personal` information,

Comment (0xTail) [reply packages/reputation/test/util.js]: yup!

File: packages/reputation/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    truffle compile
+
+## Test
+
+Run Ganache before testing:
+
+    ganache-cli --allowUnlimitedContractSize true --gasLimit 800000000 --gasPrice 1
+
+The allowUnlimitedContractSize flag gets by ganache's low byte code limit on contracts (it was small like this before on main net and test net but they increased the size a while back.  Given that, the arbitration contract rubs pretty close against the new limit). The gas limit on main net is 8 million but for some reason the gas prices for instructions are wrong on ganache and it cost way more to deploy (Arbitration.sol runs about 6-7 million gas, to clear that you'll need gas price a little higher than normal because a high gas price for 3 million as would knock you out of the deploy for a given block block).
```
Comment (woeltjen) [inline packages/reputation/README.md]: This comment references Arbitration.sol; is it applicable here?

Comment (0xTail) [reply packages/reputation/README.md]: ah thanks for catching that. removing now. thanks.

**Pull request merged**